bluepyopt/ephys/mechanisms.py

```diff
@@ -109,9 +109,9 @@ def instantiate(self, sim=None, icell=None):
                     isec,
                     sim)
 
-            logger.debug(
-                'Inserted %s in %s', self.suffix, [
-                    str(location) for location in self.locations])
+        logger.debug(
```

Comment:
If this is moved outside of the locations loop, won't it only print a message for the last insert ?

Reply:
ok, you're right. didn't see the for loop inside the string generation